TrajectoryPredictor.py

```diff
@@ -194,7 +194,8 @@ def draw_points_to_frame(self, points: np.array, frame: np.array,
         # Draw each point
         for pt in pts:
             x, y = pt[0]
-            cv2.circle(frame, (int(x), int(y)), 2, (255, 0, 0), 2)
+            if x is not None and x >= 0 and y >= 0 and x <= frame.shape[1] and y <= frame.shape[0]:
```

Comment:
Can we remove `x is not None` ? Are there cases when cv2.projectPoints returns None as an element in the output points?

Reply:
Yup!!!

Reply:
I luv u henrychan